src/server/utils/renderModuleStyles.js

```diff
@@ -16,12 +16,58 @@
 
 import { getModule } from 'holocron';
 
+/**
+ * Generates a style tag with unique ID attribute per CSS file loaded.
+ * @returns {string}
+ */
+const generateStyleTag = ({ css, digest }) => `
+<style id="${digest}" data-ssr="true">
+  ${css}
+</style>`;
```

Comment:
keep the file size as small as possible
```suggestion
const generateStyleTag = ({ css, digest }) => `<style id="${digest}" data-ssr="true">${css}</style>`;
```

Reply:
[aeae31c](https://github.com/americanexpress/one-app/pull/1099/commits/aeae31ca6e079b536f3481399044c2072942391a)